include/swift/SwiftRemoteMirror/SwiftRemoteMirrorTypes.h

```diff
@@ -48,11 +48,22 @@ typedef struct swift_reflection_section {
   void *End;
 } swift_reflection_section_t;
 
+/// Represents the remote address and size of an image.
```

Comment:
I don't think that this really describes the structure and its usage.  The address and size would be inferred from the structure itself.  What does it actually represent?  I believe that it is really a metadata table chunk.

Reply:
You're right. Maybe "/// Represents the remote address and size of an image's section."?

Reply:
Like the previous comment, I don't think that the term section should play into this.  I would say that it represents a (contiguous) region in the image's address space.